openslides/urls.py

```diff
@@ -1,52 +1,17 @@
 from django.conf.urls import include, patterns, url
+from django.views.generic import RedirectView
 
-from openslides.core.views import IndexView, ErrorView
-from openslides.users.views import UserSettingsView, UserPasswordSettingsView
+from openslides.core.views import IndexView
 from openslides.utils.rest_api import router
 
-handler403 = ErrorView.as_view(status_code=403)
```

Comment:
I talked with @emanuelschuetze about this. We both think, that the error handlers should be removed.

The only views that exist as django views are "ajax-views" or the pdf-views. So we do only need the base template for the error views. To create only a base template for this cases seems to much work.

A 404 can not happen, because all unknown urls are redirected to angular. A 403er can only happen when the user is logged in and requests a pdf which he is forbidden to see. The user can only see the 500er page on a pdf view (which should never happen). The user will never request a "ajax-view" directly.

Django has build in error pages, they are not so pretty but ok for this rare cases.